pages/corporate-training/index.tsx

```diff
@@ -0,0 +1,162 @@
+import Image from 'next/image';
+import Head from 'components/head';
+import HeroBanner from 'components/HeroBanner/HeroBanner';
+import OutboundLink from 'components/OutboundLink/OutboundLink';
+
+const pageTitle = 'Corporate Training: Breaking Biases';
+
+const CorporateTraining = () => {
+  return (
+    <>
+      <div className="relative">
+        <Head title={pageTitle} />
+        <HeroBanner
+          className="pt-0 text-center h-[750px] md:h-0"
+          backgroundImageSource="/static/images/heroImage.jpg"
+          title=""
+        />
+        <div className="absolute top-[120px] md:justify-center inset-0 flex flex-col items-center text-center">
+          <h1 className="text-white">Corporate Training: Breaking Biases</h1>
+          <p className="text-lg text-white max-w-[72%] mt-3">
+            Operation Code has provided corporate training for tech employers since 2019. In order
+            to break barriers and blockers for our military community, we must address the implicit
+            and overt biases. Reach out to us if you would like more information on how our military
+            cultural competency training works, if you&apos;d like us to help create a military
+            Diversity, Equity, Inclusion and Belonging strategy, provide ongoing professional
+            development or set up a military Employee Resource Group with you: Contact the{' '}
+            <OutboundLink
+              href="mailto:partnerships@operationcode.org"
+              analyticsEventLabel="Email"
+              hasIcon={false}
+            >
+              Partnerships Team
+            </OutboundLink>
+            . We look forward to hearing from you!
+          </p>
+        </div>
+      </div>
+      <div className="sm:block md:flex md:flex-col lg:flex-row md:items-center w-full bg-white">
```

Comment:
From this point on, there's a few things I'd like to critique:
1. We love semantic markup here and this is definitely an ordered list, so we should be using an `ol` element.
2. We should avoid arbitrary values when their are nearby (or exact), relevant values that tailwind provides. For example, `tracking-[.05em]` is simply `tracking-wider`
3. Header elements are [used by assistive technologies to navigate content quickly](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/). Having 2 `h6` elements be siblings is a small annoyance for them and of no benefit to anybody else.
4. Instead of directly coding all the markup alongside the content, it would've been a really good idea to hoist the content into a data structure to map over so that no code is being repeated.
5. For the `a` tags, please note the `OutboundLink` component!
6. It looks like we departed from the idea of the images having a square aspect ratio. At larger viewports it looks a wee bit silly, but I think we should stick closely to the design there!

Keeping that critique in mind, here's where I landed on my attempt of branching from your code:

```tsx
import Image from 'next/image';
import Head from 'components/head';
import HeroBanner from 'components/HeroBanner/HeroBanner';
import OutboundLink from 'components/OutboundLink/OutboundLink';
import { ReactNode } from 'react';
import classNames from 'classnames';

const pageTitle = 'Corporate Training: Breaking Biases';

interface Bias {
  title: string;
  subtitle: string;
  description: ReactNode;
  image: string;
  alt: string;
}

const biases: Bias[] = [
  {
    title: 'Bias #1',
    subtitle: 'All Veterans have PTSD or some form of mental illness',
    description: (
      <>
        The truth is that 28% of OIF/OEF/OND Vets self-reported one mental health diagnosis (NIH),
        and more than 1 in 5 adult civilians in the general population live with a mental illness
        (CDC).
      </>
    ),
    image: '/static/images/bias1.jpg',
    alt: 'Woman listening to someone talking',
  },
  {
    title: 'Bias #2',
    subtitle: 'Veterans are violent and unsafe to be around',
    description: (
      <>
        The truth is that TV, movies and sensationalized media propels these violent stereotypes.
        The truth: the incarceration rate for Vets is lower than for non-Vets (Bureau of Justice
        Statistics). Access to mental health resources & reducing stigma reduces the rate of
        criminalization for all.
      </>
    ),
    image: '/static/images/bias2.jpg',
    alt: 'Woman listening to someone talking',
  },
  {
    title: 'Bias #3',
    subtitle: 'Vets come from low income means, are uneducated or joined out of desperation',
    description: (
      <>
        Technological, tactical, and strategic planning focuses on raising the bar for recruits. The
        truth is the majority of new recruits come from median family income & cognitive skills,
        equal to or greater than civilians (Journal of Strategic Studies).
      </>
    ),
    image: '/static/images/bias3.jpg',
    alt: 'Woman listening to someone talking',
  },
  {
    title: 'Bias #4',
    subtitle:
      'Reserve and National Guard Service Members are part-time warriors and not able to focus on civilian careers.',
    description: (
      <>
        Reserve and National Guard members face a new challenge: bias from civilian coworkers and
        hiring managers{' '}
        <OutboundLink
          href="https://www.military.com/"
          analyticsEventLabel="Bias #4 Source Link"
          hasIcon={false}
        >
          military.com
        </OutboundLink>
        . The truth: work cultures that promote and support Reserve and NG members retain loyalty,
        increased productivity & diversity and sets an example for employees to define exemplary
        service to others.
      </>
    ),
    image: '/static/images/bias4.jpg',
    alt: 'Woman listening to someone talking',
  },
  {
    title: 'Bias #5',
    subtitle: 'Military and Veteran spouses choose not to work and enjoy being dependents',
    description: (
      <>
        There are ~ 11.5M military and Veteran spouses. The under-employment and unemployment rate
        is ~13%, over 3X the national rate{' '}
        <span className="inline-block whitespace-normal">
          [
          <OutboundLink
            href="https://www.dol.gov/sites/dolgov/files/WB/mib/WB-MilSpouse-factsheet.pdf"
            analyticsEventLabel="Bias #5 Source Link"
            hasIcon={false}
          >
            Department of Labor, Women&apos;s Bureau
          </OutboundLink>
          ]
        </span>
        . The Truth: to support our Vets, in their return to the civilian workforce, we MUST support
        their spouses and family members.
      </>
    ),
    image: '/static/images/bias5.jpg',
    alt: 'Woman listening to someone talking',
  },
];

const CorporateTraining = () => {
  return (
    <>
      <Head title={pageTitle} />

      <HeroBanner
        className="py-12"
        backgroundImageSource="/static/images/heroImage.jpg"
        title="Corporate Training: Breaking Biases"
      >
        <p className="text-left">
          Operation Code has provided corporate training for tech employers since 2019. In order to
          break barriers and blockers for our military community, we must address the implicit and
          overt biases. Reach out to us if you would like more information on how our military
          cultural competency training works, if you&apos;d like us to help create a military
          Diversity, Equity, Inclusion and Belonging strategy, provide ongoing professional
          development or set up a military Employee Resource Group with you: Contact the{' '}
          <OutboundLink
            href="mailto:partnerships@operationcode.org"
            analyticsEventLabel="Email"
            hasIcon={false}
          >
            Partnerships Team
          </OutboundLink>
          .<span className="block mt-4">We look forward to hearing from you!</span>
        </p>
      </HeroBanner>

      <ol className="flex flex-col w-full list-none">
        {biases.map(bias => (
          <li
            key={bias.title}
            className={classNames(
              'flex md:even:flex-row-reverse md:flex-row flex-col-reverse flex-wrap md:flex-nowrap md:[&>*]:flex-1',
              'even:bg-themeSecondary even:text-white', // mobile alternating colors per li
              'md:[&:nth-child(1n)]:bg-white md:[&:nth-child(2n)]:bg-themeGray800 md:[&:nth-child(3n)]:bg-themeSecondary md:[&:nth-child(1n)]:text-themeSecondary md:[&:nth-child(3n)]:text-white', // non-mobile alternating colors per li
            )}
          >
            <article className="flex flex-col items-center justify-center text-left">
              <div className="p-10 max-w-prose">
                <h3 className="flex flex-col gap-2 mb-4">
                  <span className="block text-2xl tracking-tight">{bias.title}</span>
                  <span className="tracking-wider leading-8">{bias.subtitle}</span>
                </h3>

                <p>{bias.description}</p>
              </div>
            </article>

            <div
              className={classNames(
                'relative h-64 w-64 mx-auto mt-10 mb-2 md:m-0 md:aspect-square 2xl:aspect-[4/2] md:h-auto md:w-auto',
                "before:content-[''] before:absolute before:top-2 before:left-2 before:w-full before:h-full before:bg-themePrimary", // mobile only bg square
                'md:before:content-[unset]',
              )}
            >
              <Image
                src={bias.image}
                alt={bias.alt}
                layout="fill"
                objectFit="cover"
                className="2xl:object-[75%_0%]"
              />
            </div>
          </li>
        ))}
      </ol>
    </>
  );
};

export default CorporateTraining;

```